src/twisted/conch/client/knownhosts.py

```diff
@@ -598,20 +601,19 @@ def body(ignored):
                     answer = f.readline().strip().lower()
                     if answer == b"yes":
                         return True
-                    elif answer == b"no":
+                    elif answer in {b"no", b""}:
```

Comment:
Maybe update the docstring

```diff
-'yes' and L{False} when the user answers 'no'. It may errback if
-there were any I/O errors.
+'yes' and L{False} when the user answers 'no' or no answer.
+It may errback if there were any I/O errors.